ci/nightly/pipeline.yml

```diff
@@ -480,7 +480,7 @@ steps:
     label: "Checks upgrade from previous version, whole-Mz restart"
     timeout_in_minutes: 60
     agents:
-      queue: linux-x86_64
+      queue: builder-linux-x86_64
```

Comment:
Unrelated to the change: Would it make sense to rename the agents to something like `linux-x86_64-large` or `linux-x86_64-fast` so that one does not need to have the background knowledge about what agent is "better"? Could we influence that or are these predefined by Amazon?

Reply:
No, they are our names, defined in i2:/i2/buildkite.py . You are right in general, unfortunately changing the names in i2 right now will result in major CI downtime -- all the CI machines that had the old queue name will be torn down and new machines will be started for the new queue name. At the same time, people will have all those branches using the old queue name that they will all need to rebase in order to get the CI to run again for them.